packages/docusaurus/docs/advanced/01-general-reference/error-codes.mdx

```diff
@@ -186,10 +186,16 @@ Running `yarn cache clean` will cause Yarn to remove everything inside `.yarn/ca
 
 ## YN0020 - `MISSING_LOCKFILE_ENTRY`
 
+Also known as "This package doesn't seem to be present in your lockfile; run "yarn install" to update the lockfile".
```

Comment:
Yes, I know, bad practice to restate the error message. I did this here because that way, search engines will show this higher in the results for the exact error message, instead of just the error code.